content/post/11-Eliminating-Distractions/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Eliminating Distractions"
+date: 2010-11-13T15:56:33-07:00
+draft: true
+description: This is a description
```

Comment:
This is mistake on my part. Can you delete the `description` property here and on all the future posts you add?